### PR TITLE
chore: Make `samples` be standalone

### PR DESCRIPTION
**Pull request by bitsandfoxes**
The idea is to make our `samples` be as standalone and without any automagic as possible.
This makes it easier to copy sample projects, drop provided repro-projects in there, and change the package import from a nuget package to the `.csproj`.

#skip-changelog

### CODE REVIEW

File: samples/Sentry.Samples.Console.Customized/Sentry.Samples.Console.Customized.csproj [hunk 1]
```diff
@@ -5,6 +5,39 @@
...
+    See https://docs.sentry.io/cli/releases/#creating-releases
+    -->
+    <SentryCreateRelease>true</SentryCreateRelease>
+
+    <!--
+    To associate commits with releases
+    See https://docs.sentry.io/cli/releases/#commit-integration
+    -->
+    <!--    <SentrySetCommits>true</SentrySetCommits>-->
+    <!--    <SentrySetCommitOptions>&#45;&#45;local</SentrySetCommitOptions>-->
```
Comment (jamescrosswell) [inline samples/Sentry.Samples.Console.Customized/Sentry.Samples.Console.Customized.csproj:38]: Is that supposed to be `&#45;&#45;`? Shouldn't it be `--`?

File: samples/Sentry.Samples.AspNetCore.Blazor.Server/Sentry.Samples.AspNetCore.Blazor.Server.csproj [hunk 2]
```diff
@@ -5,6 +5,15 @@
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
+    <SentryOrg>sentry-sdks</SentryOrg>
+    <SentryProject>sentry-dotnet</SentryProject>
+    <SentryUploadSymbols>true</SentryUploadSymbols>
+    <SentryUploadSources>true</SentryUploadSources>
+    <SentryCreateRelease>true</SentryCreateRelease>
```
[the remark this answers is not in the document]
Comment (bitsandfoxes) [reply samples/Sentry.Samples.AspNetCore.Blazor.Server/Sentry.Samples.AspNetCore.Blazor.Server.csproj:14]: Good idea!

File: samples/Sentry.Samples.Console.Customized/Sentry.Samples.Console.Customized.csproj (hunk 1 above)
Comment (bitsandfoxes) [reply samples/Sentry.Samples.Console.Customized/Sentry.Samples.Console.Customized.csproj:38]: You cannot have the `--` within a comment. In that case, the IDE automatically converts `--` into the character entity reference. When uncommenting, the IDE turns it back into hyphens. But it'd work either way.

Comment (jamescrosswell) [reply samples/Sentry.Samples.Console.Customized/Sentry.Samples.Console.Customized.csproj:38]: OK, bizarre. I would have assumed the escape sequence was `<!--` and so a plain old `--` would be OK but anyway, if it's not allowed then it's not allowed.

**Review by jamescrosswell — APPROVED** [review]
LGTM

**Pull request merged**